lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -188,8 +185,15 @@ export class ResponseHandler {
         // AccessToken
         const responseScopes = ScopeSet.fromString(serverTokenResponse.scope);
         // Expiration calculation
-        const expiresInSeconds = TimeUtils.nowSeconds() + serverTokenResponse.expires_in;
-        const extendedExpiresInSeconds = expiresInSeconds + serverTokenResponse.ext_expires_in;
+        const currentTime = TimeUtils.nowSeconds();
+        let tokenExpirationSeconds = serverTokenResponse.expires_in;
+        if (!libraryState) {
+            tokenExpirationSeconds += currentTime;
+        } else {
+            console.log(libraryState.ts);
```

Comment:
`console.log`

Reply:
This could be a one-line ternary as well:

```js
const timeStamp = libraryState ? libraryState.ts : currentTime;
const tokenExpirationSeconds = severTokenResponse.expires_in + timeStamp;
```

Reply:
done.